src/main/java/spark/staticfiles/StaticFiles.java

```diff
@@ -16,56 +16,135 @@
  */
 package spark.staticfiles;
 
+import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
 
-import org.eclipse.jetty.server.Handler;
-import org.eclipse.jetty.server.handler.ResourceHandler;
-import org.eclipse.jetty.util.resource.Resource;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import spark.resource.AbstractFileResolvingResource;
+import spark.resource.AbstractResourceHandler;
+import spark.resource.ClassPathResource;
+import spark.resource.ClassPathResourceHandler;
+import spark.resource.ExternalResource;
+import spark.resource.ExternalResourceHandler;
 import spark.utils.Assert;
+import spark.utils.IOUtils;
 
 /**
- * Utility methods for setting internal and external static file locations.
+ * Holds the static file configuration.
  */
 public class StaticFiles {
+    private static final Logger LOG = LoggerFactory.getLogger(StaticFiles.class);
+
+    private static List<AbstractResourceHandler> staticResourceHandlers = null;
```

Comment:
All this static state makes me uncomfortable, as you've just made a blob of non-thread-safe global mutable state, that any client can manipulate.